pkgs/development/interpreters/boron/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchgit
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "boron";
+  version = "2.0.8";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/urlan/boron/code";
+    rev = "v${version}";
+    sha256 = "sha256-RpnMs9DqcbK/gLDH5x3/DvxodJb4tR76Yk8SbYtIkV4=";
+  };
+
+  # this is not a standard `configure`
+  configurePhase = ''
+    patchShebangs .
+    ./configure
+  '';
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  buildInputs = [
+    zlib
+  ];
+
+  installTargets = "install install-dev";
```

```suggestion
  installTargets = [ "install" "install-dev" ];
```

The other format is not recommended.

---

Applied in https://github.com/NixOS/nixpkgs/pull/180865/commits/5bfa1da448c9bb6865e699a373da2f0563c106bc